compat/src/render.js

```diff
@@ -10,7 +10,7 @@ export const REACT_ELEMENT_TYPE =
 	(typeof Symbol != 'undefined' && Symbol.for && Symbol.for('react.element')) ||
 	0xeac7;
 
-const CAMEL_PROPS = /^(?:accent|alignment|arabic|baseline|cap|clip(?!PathU)|color|fill|flood|font|glyph(?!R)|horiz|marker(?!H|W|U)|overline|paint|stop|strikethrough|stroke|text(?!L)|underline|unicode|units|v|vector|vert|word|writing|x(?!C))[A-Z]/;
+const CAMEL_PROPS = /^(?:accent|alignment|arabic|baseline|cap|clip(?!PathU)|color|dominant|fill|flood|font|glyph(?!R)|horiz|marker(?!H|W|U)|overline|paint|stop|strikethrough|stroke|text(?!L)|underline|unicode|units|v|vector|vert|word|writing|x(?!C))[A-Z]/;
```

Comment:
I think this might be slightly smaller:

```suggestion
const CAMEL_PROPS = /^(?:accent|alignment|arabic|[bB]aseline|cap|clip(?!PathU)|color|fill|flood|font|glyph(?!R)|horiz|marker(?!H|W|U)|overline|paint|stop|strikethrough|stroke|text(?!L)|underline|unicode|units|v|vector|vert|word|writing|x(?!C))[A-Z]/;
```

Reply:
Hi Jason, I'm not sure this will fix the transformation of **dominantBaseline** into **dominant-baseline** (or **alignment-baseline**) as this  part of the regex only applies to the start of the string. 
I see your point of targeting only **Baseline$** instead of **^(dominant|baseline)**.
Here I presume that **^baseline** was here to manage [baseline-shift](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/baseline-shift).
We can make an attempt of dealing with those 3 attributes (dominant-baseline, alignment-baseline and baseline-shift): ```js
const CAMEL_PROPS = /^(?:accent|arabic|cap|clip(?!PathU)|color|fill|flood|font|glyph(?!R)|horiz|marker(?!H|W|U)|overline|paint|stop|strikethrough|stroke|text(?!L)|underline|unicode|units|v|vector|vert|word|writing|x(?!C))[A-Z]|[bB]aseline/
```

Reply:
This is ok for the unit test, but clearly not a win for the readability...

```suggestion
const CAMEL_PROPS = /(^(?:accent|arabic|cap|clip(?!PathU)|color|fill|flood|font|glyph(?!R)|horiz|marker(?!H|W|U)|overline|paint|stop|strikethrough|stroke|text(?!L)|underline|unicode|units|v|vector|vert|word|writing|x(?!C))[A-Z])|([bB]aseline)/;
```